piq/brisque.py

```diff
@@ -11,9 +11,114 @@
 import torch
 from torch.nn.modules.loss import _Loss
 from torch.utils.model_zoo import load_url
-import torch.nn.functional as F
+import torch.nn.functional as f
```

Comment:
It's very common to import `functional` module as `F`. Why do you want to change this?